packages/amesos2/src/Amesos2_Kokkos_View_Copy_Assign.hpp

```diff
@@ -77,36 +77,53 @@ update_dst_size(dst_t & dst, const src_t & src) {  // templated just for 2d
 template<class dst_t, class src_t> // version for same memory spaces
 typename std::enable_if<std::is_same<typename dst_t::value_type,
   typename src_t::value_type>::value>::type
-implement_copy_or_assign_same_mem_check_types(dst_t & dst, const src_t & src) {
+implement_copy_or_assign_same_mem_check_types(bool bInitialize, dst_t & dst, const src_t & src, bool & bAssigned) {
```

Comment:
Could you add a comment prior indicating what `bInitialize` and `bAssigned` represent? I know it's indicated in other usage areas as well, but I think it will help to have explanation here as well to avoid need to search

Reply:
Fixed